coap-metrics/src/main/java/com/mbed/coap/metrics/micrometer/MicrometerMetricsFilter.java

```diff
@@ -0,0 +1,101 @@
+package com.mbed.coap.metrics.micrometer;
```

Comment:
Add licence header (use IntelliJ to automatically generate one)
Lets use package `org.opencoap.coap.metrics.micrometer` as it is a totally new module.